integration-tests/test_register_with_activation_keys.py

```diff
@@ -36,15 +37,41 @@ def test_register_with_activation_keys(external_candlepin, subman, test_config,
         response = json.loads(
             private_proxy.RegisterWithActivationKeys(candlepin_config("org"), act_keynames, {}, {}, locale)
         )
-        if num_of_act_keys_to_use == 0:
-            assert "No activation key specified" in response
-        else:
-            assert (
-                "activationKeys" in response
-            ), "DBus method returns which activation keys were used to register a system"
-
-        logger.debug(response["activationKeys"])
-        assert sorted([ii["activationKeyName"] for ii in response["activationKeys"]]) == sorted(act_keynames)
+        assert (
+            "activationKeys" in response
+        ), "DBus method returns which activation keys were used to register a system"
+        assert subman.is_registered
```

Comment:
Why you changed this? It seems that it is not related to adding new test. It is also not described in commit log nor PR.

Reply:
removed assert message. a reason of failure is obvious without the message too.